README.md

```diff
@@ -7,7 +7,13 @@ This repository contains the [express middleware](https://expressjs.com/en/guide
 Check out [our website](http://avatars.adorable.io/) for more info on (and an interactive demo of) what this service does.
 
 ## How do I use it?
-First, you'll need to install the package:
+First, you'll need to install the dependencies:
+
+```bash
+brew install imagemagick
```

Comment:
This is fine, but I would maybe just link to the [imagemagick repo] and defer installation instructions to them. We can't really make many assumptions about architecture.